pkg/apis/componentconfig/types.go

```diff
@@ -163,53 +163,62 @@ type GroupResource struct {
 type KubeControllerManagerConfiguration struct {
```

Comment:
do we have a version type in k8s.io/api as well?

Note: we want to move these types into cmd/ as a follow-up.

Reply:
No, will mark it in follow-up later.